plutus-contract-certification/src/Plutus/Contract/Test/Certification/Run.hs

```diff
@@ -103,11 +107,29 @@ makeLenses ''CertificationReport
 certResJSON :: CertificationReport m -> String
 certResJSON = unpack . encode
 
-data CertificationOptions = CertificationOptions { certOptNumTests :: Int
-                                                 , certOptOutput   :: Bool }
+data CertificationEvent = QuickCheckTestEvent (Maybe Bool)  -- ^ Nothing if discarded, otherwise test result
+                        | StartCertificationTask CertificationTask
```

Comment:
Rather than start events, would it be difficult to switch to end events with a result? Then we can infer start from when the previous stage ends (or when I first call certify)

Reply:
What result are you looking for here? Would a boolean pass/fail be sufficient?

Reply:
Yes that would be great!

Reply:
Technically that information is already there in the `QuickCheckTestEven` constructor though?

Reply:
I suppose you don't get it for the unit tests though...

Reply:
I propose start and end events to make the consumer less dependent on the internals of this function. I've implemented this and I'll push it soon...